Styles/styles-beach.css

```
@@ -0,0 +1,128 @@
+@import url(styles-global.css);
+
+.header-item {
+    background-image: url(../Beach-Assets/header-image.jpg);
+}
+
+.header-text {
+    min-width: 70%;
+}
+
+.paragraph-style {
+    margin: auto;
+}
+
+:is(.image1, .image2) {
```

având în vedere că nu mai e nimic pe lângă, poate ar merge și fără `:is`.

```suggestion
.image1, .image2 {
```